src/nvim/tui/tui.c

```diff
@@ -1100,6 +1100,19 @@ static void tui_grid_scroll(UI *ui, Integer g, Integer startrow, Integer endrow,
     }
     cursor_goto(ui, top, left);
 
+    int bg;
```

Comment:
`print_attr_id` would be need to be changed. Though we could just replace the entire added block by `update_attrs(ui, 0);` which would do that as well.